connectors/file/translator-excel/src/main/java/org/teiid/translator/excel/ExcelMetadataProcessor.java

```diff
@@ -156,6 +160,22 @@ private void addTable(MetadataFactory mf, Sheet sheet, String xlsName, String or
 		Row dataRow = null;
 		int lastCellNumber = headerRow.getLastCellNum();
 		
+		// only accept cells that have a value for the name when using a header row
```

Comment:
So this is effectively truncating the table definition until the first non empty row. Is that the intention?